### PR TITLE
[#1764] Trackevent now always open on single click

### PR DESCRIPTION
**Pull request by dseif**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @benrito received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/952679fd6b2ef81/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/952679fd6b2ef81/output.txt

Total script time: 1.81 mins

Server is running on http://pr681.butyr.org/templates/test.html

File: src/editor/module.js [hunk 1]
```diff
@@ -62,38 +57,39 @@ define( [ "core/eventmanager", "core/trackevent", "./editor",
...
     butter.listen( "trackeventadded", function ( e ) {
-      var trackEvent = e.data;
+      var trackEvent = e.data,
+          view = trackEvent.view,
+          element = trackEvent.view.element;
 
       // Open a new editor on a single click
-      var trackEventMouseUp = function ( e ) {
-        if( butter.selectedEvents.length === 1 && !trackEvent.dragging ){
+      var trackEventClicked = function ( e ) {
```
Comment (ScottDowne) [inline src/editor/module.js]: Why the change from trackEventMouseUp to trackEventClicked?

File: src/editor/module.js [hunk 2]
```diff
@@ -62,38 +57,39 @@ define( [ "core/eventmanager", "core/trackevent", "./editor",
...
-      trackEvent.view.element.addEventListener( "dblclick", onTrackEventDoubleClicked, false );
+      view.listen( "trackeventdragstopped", function() {
+        element.addEventListener( "mouseup", trackEventClicked, true );
+      });
 
       butter.listen( "trackeventremoved", function ( e ) {
         if ( e.data === trackEvent ) {
-          trackEvent.view.element.removeEventListener( "mouseup", trackEventMouseUp, true );
-          trackEvent.view.element.removeEventListener( "dblclick", onTrackEventDoubleClicked, false );
+          element.removeEventListener( "click", trackEventClicked, true );
```
Comment (ScottDowne) [inline src/editor/module.js]: I am not seeing a click event being created with the trackEventClicked function.

**Comment by dseif** [comment]
Rebased

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/8b2c963338dc2b3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/8b2c963338dc2b3/output.txt

Total script time: 1.89 mins

Server is running on http://pr681.butyr.org/templates/test.html

**Comment by jbuck** [comment]
Staged on master: https://github.com/mozilla/butter/commit/e497961ad32e659ff0eafa034382bc8878139f99
